docs/json/radarr/cf/anime-lq-groups.json

```diff
@@ -501,6 +519,15 @@
         "value": "\\b(Kanjouteki)\\b"
       }
     },
+    {
+      "name": "KazeTV",
```

Comment:
not a big deal, but this should be all caps (KAZETV). Rest looks good to me.

Reply:
will change that 👍🏼